README.md

```diff
@@ -51,3 +51,111 @@ Install an OpenPGP implementation:
   to a package called `gnupg2`
 * Android: https://www.openkeychain.org/
 * iOS: https://privacyapp.io/
+
+## After the event
+
+You should now have a table with a set of validated fingerprints and ids. You
+can use this to now sign any of those identities that you have personally
+validated.
+
+This guide is written assuming you have `gpg2` installed as `gpg` if any of
+these commands fail, you can try using `gpg2` instead.
+
+### The table
+
+An example of a filled in table, noting validated fingerprints:
+
+| name                  | fingerprint                  | fp verified | id verified |
+| --------------------- | ---------------------------- | ----------- | ----------- |
+| Elizabeth Mathis      | aaaa aaaa aaaa aaaa aaaa ... | no          | n/a         |
+| Kimberly Duncan       | bbbb bbbb bbbb bbbb bbbb ... | yes         | yes         |
+| Amy Lambert           | cccc cccc cccc cccc cccc ... | yes         | no          |
+| Bill North            | dddd dddd dddd dddd dddd ... | yes         | William     |
+
+### Anatomy of a PGP identity.
+
+PGP identities are RFC2822 email address headers, they include a name, an
+optional comment and a mail address:
+
+```
+John Smith (this is a test) tagrain@example.com
+└────┬───┘ └──────┬───────┘ └────────┬────────┘
+    name       comment             mail
+```
+
+### Importing the keys.
+
+I've includes a file in this repository `./keys.asc` that contains a dump of
+every key that was submitted to this repository over the course of PyCon UK.
+Do not sign every one of these keys: some of them you may have validated, some
+of them you may not have. Only sign the keys with fingerprints that you have
+personally validated.
+
+To import the keys on the command line run:
+
+```
+gpg --import path/to/keys.asc
+```
+
+### Sigining the keys.
+
+For each of the fingerprints you have validated run:
+
+```
+gpg --sign-key --ask-cert-level --ask-cert-expire <fingerprint>
+```
+
+### Choosing a certification level.
+
+You will be given the option to choose a "certification" level this is a value
+that you should personally choose:
+
+For example:
+
+* if you have marked a fingerprint as not `fp verified` do not sign.
```

Comment:
if you have NOT marked a fingerprint as `fp verified` DO NOT SIGN

Reply:
Currently you have these a little out of order from unknown -> full verification.  I feel that having them in order of trustiness might be more intuitive.  Alternatively, maybe a diagram would be more helpful?